frontend/lib/src/components/widgets/DataFrame/columns/index.ts

```diff
@@ -63,19 +66,19 @@ export const ColumnTypes = new Map<string, ColumnCreator>(
 export const CustomCells = []
 
 export {
-  ObjectColumn,
-  TextColumn,
+  AreaChartColumn,
```

Comment:
This sorting was triggered by our formatting rules

Reply:
slightly concerning this didn't get formatted when running lintLib 🤔

Reply:
yeah, I only saw this a couple of times since you added the sorting... but not sure why some files weren't covered